wiki/Lua-Defines-module.md

```diff
@@ -80,6 +80,8 @@ _**NOTE:** Currently internal events do not expect any arguments or return value
 | N/A | ~~ON_LOAD~~ | ~~`None`~~ | ~~`None`~~ | ~~Run code after the game is loaded (currently after hyperspace.xml is initialized but might change to on main menu loading so all Lua is ready first), currently handled by `script.on_load` this internal event will potentially replace it~~ |
 | N/A | ~~PLAYERSHIP_ON_HULL_DAMAGE~~ | ~~`int damage`~~ | ~~`int` hull value~~ | ~~Detect damage to the player ship & return a value of the final hull hitpoints, returning 0 will blow up the ship. Receiving a negative value implies healing~~ |
 | 1.10.0 | WEAPON_RENDERBOX | `ProjectileFactory weapon`, `int cooldown`, `int maxCooldown`, `string firstLine`, `string secondLine` | `string firstLine`, `string secondLine` | Change the text displayed for the player weapon cooldown timers |
+| 1.11.0 | PRE_CREATE_CHOICEBOX | `LocationEvent event` | `none` | Called before the creation of a `choiceBox` by an event |
+| 1.11.0 | POST_CREATE_CHOICEBOX | `choiceBox choiceBox`, `LocationEvent event` | `none` | Called after the creation of a `choiceBox` by an event |
```

Comment:
The `ChoiceBox` struct is still refereed to with the incorrect capitalization `choiceBox`